### ISSUE TITLE
[Bug][NPU] Bilinear Resize in a small side-branch yields nondeterministic garbage on Windows NPU driver 32.0.100.4724 (correct on 32.0.100.3717)

### ISSUE THREAD

**Issue by safae33** [openvinotoolkit/openvino#36852]
### OpenVINO Version
2026.1 (also reproduced identically on 2025.0)

### Operating System / Hardware
Windows 11 (24H2) - Intel Core Ultra Series 1 (Meteor Lake), NPU "Intel AI Boost", `ov::device::architecture` = 3720

### Device / Driver
- **Broken:** Windows NPU driver **32.0.100.4724** (`ov::intel_npu::driver_version` = 1004724) - reproduced on **two independent machines**
- **Working:** Windows NPU driver **32.0.100.3717** (1003717) - same binaries, same models, fully correct

### Issue description
An FP16 image model (input `[1,1,256,256]`) contains a large segmentation branch and a small side-branch:

```
input -> Div(255) -> Resize(linear, align_corners, scales 0.25)
      -> depthwise Conv stacks (3x3 and 7x7) -> Add -> GlobalAveragePool -> Flatten -> Gemm(256->1) -> scalar logit
```

On driver 32.0.100.4724, the side-branch scalar output is **nondeterministic garbage**: magnitudes from +26 to +3400 (expected range is within about +-20), varying between compilations of the *identical* model (including with `NPU_BYPASS_UMD_CACHING=YES` and fresh `ov::Core` instances). Meanwhile the large branches of the **same compiled model** produce correct outputs (validated element-wise against CPU). CPU and GPU execution of the same model are correct. Reproduced with `NPU_COMPILER_TYPE=PLUGIN` **and** `DRIVER`, on OpenVINO **2026.1 and 2025.0**, with **two different weight sets** of the same topology.

On driver 32.0.100.3717, everything above is correct (side-branch matches CPU within normal FP16 tolerance, deterministic across compiles).

### Isolation of the trigger
Replacing **only** the `Resize` with two constant `MatMul`s implementing the same separable bilinear downsample (CPU outputs bit-identical between the two model variants) **completely fixes** the NPU output on driver 32.0.100.4724 - same machines, same driver, same runtime, deterministic, matches CPU within ~0.01. This isolates the defect to the lowering/codegen of `Resize` (linear, align_corners, fixed 0.25 scales) in this small-branch topology.

Note: per the 32.0.100.4724 release notes, OpenVINO 2026.1 is the validated framework version for this driver, so the failing configuration is inside the validated matrix.

### Secondary issue (same driver family)
With the OpenVINO **2025.0** NPU plugin, `ov::Core::get_available_devices()` hard-crashes with an access violation (0xC0000005) inside the NPU plugin on driver 32.0.100.4724 (compilation and inference work if enumeration is skipped). Similar to previously reported enumeration crashes.

### Reproduction
We can provide on request: (1) a sanitized model with randomized weights reproducing the topology, (2) full diagnostic logs from broken and working machines (per-variant logits, compiled-model properties, loaded-module inventory, Level Zero driver versions). The affected production model is proprietary and cannot be shared as-is.

### Expected behavior
The side-branch output should match CPU within FP16 tolerance and be deterministic across compilations, as it is on driver 32.0.100.3717.

### Credits
Thanks to @umutdundar99 for isolating the trigger and implementing the MatMul-based model workaround.

**Comment by Zulkifli-Intel** [comment]
Hi @safae33 

To help us investigate this issue, could you please provide:
1. The sanitized model with randomized weights replicating the topology.
2. The Inference script to run with the model.
3. The full diagnostic logs you mentioned (per-variant logits, loaded-module inventory, Level Zero driver versions).

**Comment by Zulkifli-Intel** [comment]
Closing due to inactivity, please create a new issue if additional assistance is needed.
